src/components/loadable.cjsx

```diff
@@ -49,5 +50,27 @@ module.exports = React.createClass
       render()
     else if isFailed()
       renderError()
+      
     else
       render()
+
+
+RefreshButton = React.createClass
+  displayName: 'RefreshButton'
+
+  reloadPage: ->
+    location.reload()
+
+  render: ->
+    if React?
```

Comment:
I wondered that myself.  Was going off of pivotal checklist:

"check if reload() on click would work (and backup browser link in case React has crashed)"